.circleci/config.yml

```diff
@@ -109,6 +109,9 @@ libressl_250: &LIBRESSL_250
 libressl_28x: &LIBRESSL_28x
   LIBRARY: libressl
   VERSION: 2.8.2
+libressl_290: &LIBRESSL_290
```

Comment:
Could you replace the 28x tests with 290? There's also the test list down all the way at the bottom of this file that you'll need to update.

Reply:
Happy to do so! Hopefully that new commit fixes this.